src/Illuminate/Routing/Route.php

```diff
@@ -360,7 +360,7 @@ public function parameter($name, $default = null)
      *
      * @param  string  $name
      * @param  mixed  $default
-     * @return string
+     * @return string|null
```

Comment:
actually, this is mixed, since default has type mixed. We should either change default to string|null, or change the return type to mixed. // cc @driesvints

Reply:
I think `string|null` is fine here and we should update `$default` to `string|null`.

Reply:
I agree with @driesvints, we expect to return a `string` or `null`.

Reply:
You agree but didn't fix the code?

Reply:
Oops! Sorry, just woke up, I was waiting for your comments yesterday.